RIOT-OS/RIOT
# doc/doxygen: use DuckDuckGo for search

### Contribution description

The Doxygen search is pretty broken, so use DuckDuckGo instead.


### Testing procedure

`make doc` and try the search (CircleCI should provide a [doc build](https://output.circle-artifacts.com/output/job/718c4dfb-6ff3-4aaf-bd4d-93ee14893b33/artifacts/0/doc/index.html))


### Issues/PRs references
